tools/run_tests/run_interop_tests.py

```diff
@@ -179,10 +182,10 @@ def global_env(self):
     return {}
 
   def unimplemented_test_cases(self):
-    return _SKIP_COMPRESSION
+    return _SKIP_ADVANCED + _SKIP_COMPRESSION
```

Comment:
_SKIP_ADVANCED shouldn't be included here (they were recently enable for Java, maybe this is from a stale source file)